testsuite/end2end/profiles.yml

```diff
@@ -338,13 +402,18 @@
     - CIM_SystemDevice
   doc: SMI-S Common Profiles
 -
-  # TODO 2018-12 AM: Clarify the scoping path of this profile.
+  # Note: Special case: This profile has two scoping paths, see below.
   registered_org: SNIA
   registered_name: Enhanced Zoning and Enhanced Zoning Control
   autonomous: false
   central_class: CIM_ZoneService
   scoping_class: CIM_AdminDomain
-  scoping_path: null
+  scoping_path:
```

Comment:
If we are to generalize this file in the future would we not be better off to keep the whole scoping_path together and make it a list of lists.

Reply:
What do you mean with keeping it together? In today's DMTF profile approach, the scoping path is an attribute of the scoped profile. I could imagine moving it to the scoping profile, for each profile it references. That would allow having different scoping paths for the same scoped profile, depending on which ptofile references it.

Reply:
you have

scoping_path:
    - CIM_HostedService
  scoping_path2:
    - CIM_HostedService

if there were three alternatives it would be:
scoping_path:
  -  ...
scoping_path1:
  -  ...
scoping_path3:
  - ...
n separate named lists.  Tonight not sure how to express it but in python it would be
[ ['CIM_HostedService'], [] ]

Reply:
Karl, Ok now I get it. I don't want to have alternatives, because these are not really alternatives but instead from a perspective of a referencing profile, there is only one correct way. It just becomes more than one from a perspective of the referenced profile because the two profiles referencing it assume a different scoping path.

That's why I brought up the idea should we not attach the scoping path to the referencing profile. For now, just for our profile definition here in pywbem.